app/version/version.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 // version a string since it is overwritten at build-time with the git tag for official releases.
-var version = "v0.17-rc"
+var version = "v0.17-rc2"
```

Comment:
For release candidates, we don't bump the `version` string. The `rc2` candidate is actually `git tagged`